contribs/gmf/apps/desktop/index.html

```diff
@@ -152,6 +192,7 @@
         module.constant('langUrls', langUrls);
         module.constant('authenticationBaseUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi');
         module.constant('fulltextsearchUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi/fulltextsearch?limit=30&partitionlimit=5');
+        module.constant('gmfAltitudeUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi/raster');
```

Comment:
Okay...  we need to discuss about that. Because Altitude service must also take another name. And even the elevation directive should be renamed. 
In all cases, I'll rename the constant directly after this PR (as discussed).